### ISSUE TITLE
Bot is crashing randomly

### ISSUE THREAD

**Issue by Plastix** [bcbwilla/nimbus#3]
```
Traceback (most recent call last):
  File "nimbus.py", line 71, in <module>
    main(config_filename)
  File "nimbus.py", line 62, in main
    text = event['text']
KeyError: 'text'
```

I can't tell what this is from.

**Comment by Plastix** [comment]
When would a slack event not have a text component?

Quick fix would be `text = event.get('text', '')`

**Comment by bcbwilla** [comment]
Currently we're only processing messages, so I'm not sure how a message would not have text. For now that fix looks fine, or we could do something like

```
if 'text' in event.keys():
    process_message(event['text'],...)
```

to avoid trying to process empty strings

**Comment by Plastix** [comment]
Is fixed in #6
